standard/standard_3.php

```diff
@@ -790,7 +795,11 @@ function putenv(string $assignment): bool {}
  * @return string[]|false[]|false This function will return an array of option / argument pairs or false on
  * failure.
  */
-function getopt(string $short_options, array $long_options, &$rest_index): array|false {}
+function getopt(
+    string $short_options,
+    array $long_options,
+    #[PhpStormStubsElementAvailable(from: '7.1')] &$rest_index
```

Comment:
Small note: since 8.0 `$rest_index`  has a default value of `null`: https://3v4l.org/gFr1v#v800 & https://www.php.net/manual/en/function.getopt.php

Reply:
Hm..That's true, but let's leave it as is as currently it's marked as optional using phpdoc for all PHP versions and replacing it with default value we can introduce false positive inspection warnings "Redundant parameter" for versions < 8.0.